doc/tutorials/arrays.md

```diff
@@ -62,9 +62,11 @@ To create an **array element** we start the basename of a key with the `#` chara
 
 ```sh
 kdb set user:/tests/sequence/#0 'First Element'
+#> Create a new key user:/tests/sequence/#0 with string "First Element"
 kdb set user:/tests/sequence/#1 'Second Element'
-# Arrays do not need to be contiguous
```

Comment:
Why do you remove this comment? It seems useful to me.

Reply:
I must have removed it unintentionally. I'll bring it back